c10/core/impl/PyInterpreter.cpp

```diff
@@ -77,6 +77,10 @@ struct NoopPyInterpreterVTable final : public PyInterpreterVTable {
   void trace_gpu_device_synchronization() const override {}
   void trace_gpu_stream_synchronization(uintptr_t stream) const override {}
   void trace_gpu_event_synchronization(uintptr_t event) const override {}
+  void trace_kernel_launch(
+      const c10::OperatorHandle& op,
+      std::vector<c10::IValue>& inputs,
```

Comment:
Squash this with the next PR, there's no usage of this here, best to keep it together